conans/server/store/server_store.py

```diff
@@ -0,0 +1,267 @@
+from os.path import normpath, join, relpath
+
+from conans.errors import NotFoundException, ConanException
+from conans.model.ref import PackageReference, ConanFileReference
+from conans.paths import SimplePaths, EXPORT_FOLDER, PACKAGES_FOLDER
+from conans.server.revision_list import RevisionList
+
+REVISIONS_FILE = "revisions.txt"
+
+
+class ServerStore(SimplePaths):
+
+    def __init__(self, revisions_enabled, storage_adapter):
+        super(ServerStore, self).__init__(storage_adapter.base_storage_folder())
+        self._revisions_enabled = revisions_enabled
```

Comment:
Maybe a hierarchy of ServerStore with and without revisions would be a bit more maintainable. But I am ok as-is.